expenses/tests_expenses.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from django.utils import timezone
+from .models import Expenses
+
+AMOUNT = '100'
+DATE = timezone.now()
+CATEGORY = 'CLOTHING'
+
+
+@pytest.fixture
+def expense_test0():
+    return Expenses(amount=AMOUNT,
+                    date=DATE,
+                    Category=CATEGORY
+                    )
```

Comment:
Maybe add tests regarding the types and values of an Expense? 
What happens if you create an expense with a non-valid category? 
Does the date of an expense count valid if it is the future? 
Can the amount be negative? 
All those and much more should be tested and validated

Reply:
> Maybe add tests regarding the types and values of an Expense? What happens if you create an expense with a non-valid category? Does the date of an expense count valid if it is the future? Can the amount be negative? All those and much more should be tested and validated

There is a default category - other.
I'm taking care about of the others
@adiabramovitch

Reply:
For now, all those things are expressed in the model. 
If we see it’s important, in the future, we will add it.